packages/reports/src/sass/app/modules/contract.scss

```diff
@@ -35,14 +35,35 @@ $CONTRACT_INFO_BOX_PADDING: 1.6em;
                     margin-right: 0.8rem;
                 }
             }
+            &-wrapper {
+                max-width: 35rem;
+
+                @include mobile {
+                    max-width: 23.3rem;
+
+                    a {
+                        font-size: 1rem;
+                    }
+                }
+
+                a {
+                    color: var(--brand-red-coral);
+                }
+            }
             &-text {
-                max-width: 360px;
+                &--collapsed {
+                    -webkit-box-orient: vertical;
+                    -webkit-line-clamp: 3;
+                    display: -webkit-box;
+                    line-clamp: 3;
```

Comment:
browser support probably

Reply:
Yeah, this is to cater different browsers, since some browsers have native support for line-clamp, while older browsers can use this style only if it has `-webkit` before.

Reply:
Yep, I see, just tried to input it in auto prefix app and got the same result